_includes/config/es-elasticsearch-magento.md

```diff
@@ -1,102 +1,96 @@
+{:style="table-layout:auto;"}
+
+{:start="4"}
+1. Click **Test Connection**.
+
+You will see either:
+
+<img src="{{ site.baseurl }}/common/images/elastic_test-success.png">
+
+Magento successfully connected to the Elasticsearch server.
+Continue with:
+
+* [Configure Apache and Elasticsearch]({{ site.gdeurl21 }}config-guide/elasticsearch/es-config-apache.html)
+* [Configure nginx and Elasticsearch]({{ site.gdeurl21 }}config-guide/elasticsearch/es-config-nginx.html)
+
+or you will see:
+
+<img src="{{ site.baseurl }}/common/images/elastic_test-fail.png">
+
+If so, try the following:
+
+* Make sure the Elasticsearch server is running.
+* If the Elasticsearch server is on a different host from Magento, log in to the Magento server and ping the Elasticsearch host. Resolve network connectivity issues and test the connection again.
+* Examine the command window in which you started Elasticsearch for stack traces and exceptions. You must resolve those before you continue.
+In particular, make sure you started Elasticsearch as a user with `root` privileges.
+* Make sure that [UNIX firewall and SELinux]({{ site.gdeurl21 }}config-guide/elasticsearch/es-overview.html#firewall-selinux) are both disabled, or set up rules to enable Elasticsearch and Magento to communicate with each other.
+* Verify the value of the **Elasticsearch Server Hostname** field. Make sure the server is available. You can try the server's IP address instead.
+* Use the command `netstat -an | grep **listen-port**` command to verify that the port specified in the **Elasticsearch Server Port** field is not being used by another process.
+  For example, to see if Elasticsearch is running on its default port, use the following command:
+  `netstat -an | grep 9200`
+  If Elasticsearch is running on port 9200, it displays similar to the following:
+  `tcp        0      0 :::9200            :::*          LISTEN`
 
 ### Reindexing catalog search and refreshing the full page cache {#es-reindex}
 
 After you change Magento's Elasticsearch configuration, you must reindex the catalog search index and refresh the full page cache using the Admin or command line.
 
 To refresh the cache using the Admin:
 
-1.  In the Admin, click <strong>System</strong> > <strong>Cache Management</strong>.
-2.  Select the checkbox next to <strong>Page Cache</strong>.
-3.  From the <strong>Actions</strong> list in the upper right, click <strong>Refresh</strong>.<br />
-		The following figure shows an example.<br />
-		![]({{ site.baseurl }}/common/images/solr_refresh-cache.png){:width="600px"}
+1. In the Admin, click **System** > **Cache Management**.
+1. Select the checkbox next to **Page Cache**.
+1. From the **Actions** list in the upper right, click **Refresh**.
+   The following figure shows an example.
+   ![]({{ site.baseurl }}/common/images/solr_refresh-cache.png){:width="600px"}
```

Comment:
Have an architect review this graphic. We no longer support solr

Reply:
Not solr related..just the file name. Admin looks the same.